ci/scripts/run_benchmarks.sh

```diff
@@ -30,7 +30,16 @@ done
 SCRIPTDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 source concourse-metadata-resource/concourse_metadata
-CLUSTER_TAG="${BUILD_PIPELINE_NAME}-${BUILD_JOB_NAME}-${BUILD_NAME}-${BUILD_ID}"
+
+if [ -z "${WITH_SSL}" ]; then
+  SSL_TAG_POSTFIX=""
+  SSL_TEST_OPTION=""
+else
+  SSL_TAG_POSTFIX="-wSSL"
+  SSL_TEST_OPTION="--tests=*tBenchmark"
```

Comment:
You can also filter Put and Get benchmarks this way:
--tests *GetBenchmark --tests *PutBenchmark